packages/worker/src/tests/structures/configs.ts

```diff
@@ -62,8 +62,8 @@ export function smtpEthereal(): SMTPConfig {
       from: "testfrom@example.com",
       secure: false,
       auth: {
-        user: "wyatt.zulauf29@ethereal.email",
-        pass: "tEwDtHBWWxusVWAPfa",
+        user: "mortimer.leuschke@ethereal.email",
```

Comment:
What's the story with this change?

Reply:
The other ethereal email / pass we were using for testing just wasn't working anymore - the account didn't exist so I created a new one for the test config